src/main/java/com/nearsoft/referrals/model/User.java

```diff
@@ -0,0 +1,66 @@
+package com.nearsoft.referrals.model;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name = "users")
+public class User implements Serializable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+    private String email;
```

Comment:
qué longitud tienen los campos varchar a los que no les específicas explícitamente su longitud con @Column?

Reply:
Tienen una longitud de 255 por lo que el error me arroja
 ERROR: value too long for type character varying(255)